DataCollection/src/Domain/Entities/ISurvey.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace FlightNode.DataCollection.Domain.Entities
+{
+    public interface ISurvey
```

Comment:
Created this interface so that some methods could interact with either `SurveyPending` or `SurveyCompleted` without caring which type it is.